Automattic/wp-calypso
# Editing Toolkit: Update to 2.8.3

### Changes proposed in this Pull Request

* Version bump and changelog

### Testing instructions

* Editor close button should deal with situation where experimental GB feature is unavailable (https://github.com/Automattic/wp-calypso/pull/46666)
* Premium Content: Remove double banners (https://github.com/Automattic/wp-calypso/pull/46659)
* Premium Content: Add missing migrations for margin fix (https://github.com/Automattic/wp-calypso/pull/46924)
* Editing Toolkit: Add a workaround to prevent slider blocks breaking the editor at full width (https://github.com/Automattic/wp-calypso/pull/46951)
* Focused Launch: added WIP Focused Launch modal behind a flag (https://github.com/Automattic/wp-calypso/pull/46686)
